### PR TITLE
divide up creation/population of temp pet table [VS-48]

### PR DESCRIPTION
**Pull request by rsasch**
Update GvsPrepareCallset step so that, instead of creating and inserting all the data into the `_pet_new` table in one step (which errors out with large callsets), add the data in smaller sections that correspond to each `pet_` in the GVS.

Closes https://broadworkbench.atlassian.net/browse/VS-48

### CODE REVIEW

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py [hunk 1]
```diff
@@ -193,34 +193,36 @@ def create_position_table(fq_temp_table_dataset, min_variant_samples):
...
-    partition_samples = get_samples_for_partition(sample_ids, i)
+    partition_samples = get_samples_for_partition(sample_ids, i)  #sample ids for the partition
 
-      # KCIBUL -- grr, should be fixed width
-    fq_pet_table = f"{fq_pet_vet_dataset}.{PET_TABLE_PREFIX}{i:03}"
     if len(partition_samples) > 0:
+      subs = {}
+      create_or_insert = f"\nCREATE OR REPLACE TABLE `{fq_temp_table_dataset}.{PET_NEW_TABLE}` {TEMP_TABLE_TTL} AS \n WITH \n" if i == 1 \
+      else f"\nINSERT INTO `{fq_temp_table_dataset}.{PET_NEW_TABLE}` \n WITH \n"
+      fq_pet_table = f"{fq_pet_vet_dataset}.{PET_TABLE_PREFIX}{i:03}"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:207]: if the \n's are all for readability, would it make sense to space them that way in this code? like end each line here with \n ?

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py [hunk 2]
```diff
@@ -193,34 +193,36 @@ def create_position_table(fq_temp_table_dataset, min_variant_samples):
...
-        "with\n" + \
-        ("\n".join(subs.values())) + "\n" \
-        "q_all AS (" + (" union all ".join([ f"(SELECT * FROM q_{id})" for id in subs.keys()]))  + ")\n" + \
-        f" (SELECT * FROM q_all)"
+      sql = create_or_insert + ("\n".join(subs.values())) + "\n" + \
+      "q_all AS (" + (" union all ".join([ f"(SELECT * FROM q_{id})" for id in subs.keys()]))  + ")\n" + \
+      f" (SELECT * FROM q_all)"
 
-  print(sql)
-  print(f"PET Query is {utf8len(sql)/(1024*1024)} MB in length")
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:220]: were we printing this before to make sure our query wasn't too long?

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:207]: does there need to be a name after AS ?

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py (hunk 2 above)
Comment (rsasch) [reply scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:220]: good catch — I'll put that back in

File: scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:207]: Both the line returns and the "AS" with no alias were copied from the original query and are in other queries.  When I added an alias it failed, so I'm going to keep this one that way.

Comment (RoriCremer) [reply scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py:207]: huh! well I can't argue with that logic....

**Pull request merged**